src/test/java/subway/line/LineAcceptanceTest.java

```diff
@@ -0,0 +1,177 @@
+
+    @BeforeEach
+    void setUpClass() {
+        Map<String, String> params = new HashMap<>();
+        params.put("name", "강남역");
+        Long firstId = stationApiCaller.callCreateStation(params).jsonPath().getObject("id", Long.class);
+        params.put("name", "삼성역");
+        Long secondId = stationApiCaller.callCreateStation(params).jsonPath().getObject("id", Long.class);
+        params.put("name", "선릉역");
+        Long thirdId = stationApiCaller.callCreateStation(params).jsonPath().getObject("id", Long.class);
+
+        newBunDangLineParams = new HashMap<>();
+        newBunDangLineParams.put("name", "신분당선");
+        newBunDangLineParams.put("color", "bg-red-600");
+        newBunDangLineParams.put("upStationId", firstId.toString());
+        newBunDangLineParams.put("downStationId", secondId.toString());
+        newBunDangLineParams.put("distance", "10");
+
+        zeroLineParams = new HashMap<>();
+        zeroLineParams.put("name", "0호선");
+        zeroLineParams.put("color", "bg-red-100");
+        zeroLineParams.put("upStationId", firstId.toString());
+        zeroLineParams.put("downStationId", thirdId.toString());
+        zeroLineParams.put("distance", "10");
+    }
+
+    /**
+     * When 지하철 노선을 생성하면
+     * Then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+    @DisplayName("지하철노선을 생성한다.")
+    @Test
+    void createLine() {
+        // when
+        ExtractableResponse<Response> response = lineApiCaller.callApiCreateLines(newBunDangLineParams);
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        // then
+        response = lineApiCaller.callApiFindLines();
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        List<String> actual = response.jsonPath().getList("name", String.class);
+        String expected = "신분당선";
+        assertThat(actual).containsAnyOf(expected);
+    }
+
+    /**
+     * Given 2개의 지하철 노선을 생성하고
+     * When 지하철 노선 목록을 조회하면
+     * Then 지하철 노선 목록 조회 시 2개의 노선을 조회할 수 있다.
+     */
+    @DisplayName("지하철노선들의 목록을 조회한다.")
+    @Test
+    void findLines() {
+        // given
+        ExtractableResponse<Response> response = lineApiCaller.callApiCreateLines(newBunDangLineParams);
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+        response = lineApiCaller.callApiCreateLines(zeroLineParams);
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.CREATED.value());
+
+        // when
+        response = lineApiCaller.callApiFindLines();
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.OK.value());
+        List<String> actual = response.jsonPath().getList("name", String.class);
+
+        // then
+        List<String> expected = List.of("신분당선", "0호선");
+        int actualSize = actual.size();
+        int expectedSize = 2;
+        assertThat(actual).isEqualTo(expected);
+        assertThat(actualSize).isEqualTo(expectedSize);
```

Comment:
containsExactly 메서드로 검증해 보셔도 좋을 것 같아요
```suggestion
        assertThat(actual).containsExactly("신분당선", "0호선");
```